frontend/src/components/Pages/Dashboard/UploadVideo.jsx

```
@@ -0,0 +1,561 @@
+          </label>
+        </div>
+      );
+    }
+    
+    return (
+      <div className="relative rounded-lg overflow-hidden bg-gray-900">
+        <video 
+          src={videoPreview} 
+          controls 
+          className="w-full h-auto max-h-64"
+        />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)